check-version-bump.sh

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env bash
```

Comment:
That script should also work on my machine ie. before pushing to the CI (it should actually be included in a makefile target).